### PR TITLE
Add config for production recon service

### PR DESCRIPTION
**Pull request by shifucun**

### CODE REVIEW

File: deploy/gke/recon-prod.yaml [hunk 1]
```diff
@@ -0,0 +1,24 @@
+# Copyright 2022 Google LLC
```
Comment (pradh) [inline deploy/gke/recon-prod.yaml:1]: do we need recon-autopush?

**Review by pradh — APPROVED** [review]
Nice!

File: deploy/gke/recon-prod.yaml (hunk 1 above)
Comment (shifucun) [reply deploy/gke/recon-prod.yaml:1]: Yep, we do need that. Have some issue on deploying it. Will send a PR when it's ready.

**Comment by shifucun** [comment]
Thanks!

**Pull request merged**